src/site/content/en/blog/css-web-vitals/index.md

```diff
@@ -0,0 +1,484 @@
+
+Before:
+
+```css
+background: url("https://cdn.pixabay.com/photo/2015/07/15/06/14/gradient-845701\_960\_720.jpg")
+```
+
+After:
+
+```css
+background: linear-gradient(135deg, #fbc6ff 20%, #bdfff9 90%);
+```
+
+
+### Images and layout shifts
+
+Browsers can only determine the size of an image once the image loads. If the
+image load occurs after the page has been rendered, but no space has been
+reserved for the image, a layout shift occurs when the image appears. In the
+demo, the hero image is causing a layout shift when it loads.
+
+{% Aside %}
+The phenomenon of images causing layout shifts is more obvious in
+situations where images are slow to load - for example, on a slow connection or
+when loading an image with a particularly large file size.
+{% endAside %}
+
+#### Identify
+
+To identify images without explicit `width` and `height`, use Lighthouse's
+"Image elements have explicit width and height" audit.
+
+
+
+{% Img
+    src="image/j2RDdG43oidUy6AL6LovThjeX9c2/wDGRVi7JaUOTjD9ODOk9.png",
+    alt="Lighthouse's 'Image elements have explicit width and height' audit",
+    width="800",
+    height="274"
+%}
+
+
+In this example, both the hero image and article image are missing `width` and
+`height` attributes. 
+
+
+#### Fix
+
+Set the `width` and `height` attributes on these images to avoid layout shifts.
+
+Before:
+
+```html
+<img src="https://source.unsplash.com/random/2000x600" alt="image to load in">
+<img src="https://source.unsplash.com/random/800x600" alt="image to load in">
+```
+
+After:
+
+```html
+<img src="https://source.unsplash.com/random/2000x600" width="2000" height="600" alt="image to load in">
+<img src="https://source.unsplash.com/random/800x600" width="800" height="600" alt="image to load in">
+```
+
+
+<figure class="w-figure">
+  {% Video src="video/j2RDdG43oidUy6AL6LovThjeX9c2/fLUscMGOlGhKnNHef2py.mp4" %}
+  <figcaption class="w-figcaption">
+    The hero image now loads without causing a layout shift.
+  </figcaption>
+</figure>
+
+
+{% Aside %}
+Another approach to loading images is to use the
+[`srcset`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/srcset)
+and
+[`sizes`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/sizes)
+attributes in conjunction with specifying `width` and `height` attributes. This
+has the additional performance advantage of allowing you to serve different
+sized images to different devices. For more information, see [Serve responsive
+images](/serve-responsive-images/).
+{% endAside %}
+
+
+## Fonts
+
+Fonts can delay text rendering and cause layout shifts. As a result, it is
+important to deliver fonts quickly.
+
+### Delayed text rendering
+
+By default, a browser will not immediately render a text element if its
+associated web fonts have not loaded yet. This is done to prevent a ["flash of
+unstyled text (FOUT)"](https://en.wikipedia.org/wiki/Flash_of_unstyled_content).
```

Comment:
```suggestion
associated web fonts have not loaded yet. This is done to prevent a ["flash of
unstyled text" (FOUT)](https://en.wikipedia.org/wiki/Flash_of_unstyled_content).
```